trade-finance-manager-ui/server/test-mocks/amendment-test-mocks.js

```diff
@@ -45,6 +45,28 @@ const session = {
 };
 
 const MOCK_AMENDMENT = {
+  amendmentId: '12345',
+  facilityId: '45678',
+  dealId: '999',
+  ukefFacilityId: '123',
+  type: CONSTANTS.FACILITY.FACILITY_TYPE.CASH,
+  status: CONSTANTS.AMENDMENTS.AMENDMENT_STATUS.IN_PROGRESS,
+  submittedByPim: true,
+  requireUkefApproval: true,
+};
+
+const MOCK_AMENDMENT_AUTOMATIC_APPROVAL = {
+  amendmentId: '12345',
+  facilityId: '45678',
+  dealId: '999',
+  ukefFacilityId: '123',
+  type: CONSTANTS.FACILITY.FACILITY_TYPE.CASH,
+  status: CONSTANTS.AMENDMENTS.AMENDMENT_STATUS.IN_PROGRESS,
```

Comment:
if it's submitted by PIM and it does not require UKEF approval, then the status should be `Completed`
Also, can we inherit the properties from `MOCK_AMENDMENT` and override anything that's different?

Reply:
done